### DISCUSSION TITLE
Unauthorized Error

### DISCUSSION THREAD

**Discussion by AkkafaOguz** [usebruno/bruno#734] [Q&A]
Hi,

I've tried to run a collection via CLI but I got (401 Unauthorized) error. Although the authorization key is at the environment file and it's possible to run folder from application without an (401 Unauthorized) error, it's not possible to run it via CLI. May I know the root cause of the problem or a way to solve this issue? Thanks.

Note: --env-var  option is also tried to overwrite authorization key but it also did not work.

**Comment by helloanoop** [comment]
This may be a bug.

I am assuming you are header is something like `Bearer {{process.env.TOKEN}}`
And you `.env` file has 
```env
TOKEN=secret
```

You can console log your header in the pre request script
```javascript
console.log(req.getHeader("Authorization"));
```

And check what's happening.
Let me know what you see. Need a bit more details to reproduce and fix the bug
